_non-cosmos/binancesmartchain/assetlist.json

```diff
@@ -174,6 +174,34 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/purse-token-logo.svg"
         }
       ]
+    },
+    {
+      "description": "The SRCX token of Source Protocol.",
+      "denom_units": [
+        {
+          "denom": "0x454b90716a9435e7161a9aea5cf00e0acbe565ae",
+          "exponent": 0
+        },
+        {
+          "denom": "srcx",
+          "exponent": 9
+        }
+      ],
+      "type_asset": "erc20",
+      "address": "0x454b90716a9435e7161a9aea5cf00e0acbe565ae",
+      "base": "0x454b90716a9435e7161a9aea5cf00e0acbe565ae",
+      "name": "Source Token",
+      "display": "SRCX",
```

Comment:
please lowercase the display denom so that it's pointing to a defined denom_unit